java/java-hello-world/kubernetes-manifests/hello.deployment.yaml

```diff
@@ -19,4 +19,6 @@ spec:
         - containerPort: 8080
         env:
         - name: PORT
-          value: "8080"
\ No newline at end of file
+          value: "8080"
+        - name: JAVA_TOOL_OPTIONS
```

Comment:
we need to add a comment that explains the purpose and what happens when folks want to remove it